packages/terra-date-input/src/DateInput.jsx

```diff
@@ -442,6 +468,38 @@ class DateInput extends React.Component {
     }
   }
 
+  /**
+   * Sets state to new value dependent on existing date
+   * @param {Object} event Event object generated from the event delegation.
+   * @param {Number} addDays Adds days to current date or today's date.
```

Comment:
I see that setHotKeyDay will set the date to today if addDays is zero. You should mention that in this param's doc since it's special, and surprising (though valid), behavior.

Reply:
Added sentence caveat for case 0: https://github.com/cerner/terra-framework/pull/1483/commits/2a6582f14d8b118be226db1e1816aa5418005e26#diff-ea35ec190c52db171ef5f774b071e80db0b038abff0e19580b310959fc582329R474